mobile/src/screens/projects/card-item/index.tsx

```diff
@@ -18,6 +18,8 @@ const CardItem: FC<CardItemProps> = ({
   return (
     <Card style={cardStyles.mainView}>
       <Card.Cover
+        height
+        width
```

Comment:
what are these?

if they are booleans, then we should pass them like this:
```suggestion
        height={true}
        width={true}
```

I will make a note to enforce this using [react/jsx-boolean-value](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md)

Reply:
They're actually unknowns but they are not optionals thus some ts errors were thrown from this component, do you want to enforce them to be equal to true?

Reply:
that will do, we will later update `eslint` to enforce that rule